app/views/downloads/_download_preface.html.erb

```diff
@@ -39,29 +40,29 @@ hide_freeform?(@work)) %>
     <p><%= ts('Summary') %></p>
     <blockquote class="userstuff"><%=raw sanitize_field(@work, :summary) %></blockquote>
   <% end %>
-  
+
   <% unless @work.notes.blank? && @work.endnotes.blank? %>
     <p><%= ts('Notes') %></p>
     <% unless @work.notes.blank? %>
       <blockquote class="userstuff"><%=raw sanitize_field(@work, :notes) %></blockquote>
     <% end %>
     <% unless @work.endnotes.blank? %>
       <div class="endnote-link">
-        <%= ts("See the end of the work for") %> <% unless @work.notes.blank? %><%= ts("more") %><% end 
+        <%= ts("See the end of the work for") %> <% unless @work.notes.blank? %><%= ts("more") %><% end
 %> <%= link_to ts("notes"), "#endnotes" %>
       </div>
-    <% end %> 
+    <% end %>
   <% end %>
 
   <% unless @work.parents.blank? %>
     <dl>
       <dt><%= ts('This work was inspired by') %></dt>
       <dd>
-        <%= @work.parent_work_relationships.map{|rw| "#{link_to(rw.parent.title, 
+        <%= @work.parent_work_relationships.reject{|x| x.parent.nil?}.map{|rw| "#{link_to(rw.parent.title,
```

Comment:
Just for the sake of making this a bit shorter, what if we did the same thing here that we did with the series list above? That is:
```
<% parent_work_list = @work.parent_work_relationships.reject{ |wr| wr.parent.nil? } %>
<%= parent_work_list.map{|rw| "#{link_to(rw.parent.title,
```
Just a thought, though!